lxd/apparmor/qemuimg.go

```diff
@@ -23,7 +23,15 @@ profile "{{ .name }}" flags=(attach_disconnected,mediate_deleted) {
   capability dac_read_search,
   capability ipc_lock,
 
-  /sys/devices/**/block/*/queue/max_segments  r,
+  /sys/devices/**/block/*/queue/max_segments r,
+  /sys/devices/**/block/*/queue/zoned r,
+  /sys/devices/system/node/ r,
+  /sys/devices/virtual/nvme-subsystem/**/queue/max_segments r,
+  /sys/devices/virtual/nvme-subsystem/**/queue/zoned r,
+  /sys/devices/system/node/*/meminfo r,
```

Comment:
Mhh this looks redundant to `/sys/devices/system/node/ r`. I have added them one after another as they appeared but maybe this one can be removed in favor of the first line.

Reply:
Ran the test suite another time without `/sys/devices/system/node/ r` and the warning seems to be gone. So I have removed it also from this PR.

Reply:
Saw it again so I'll add it back in 
![Screenshot from 2024-06-05 09-39-40](https://github.com/canonical/lxd/assets/6641364/1dc44f39-91c5-4a1f-b4fc-7c45a829bd54)

Reply:
`/sys/devices/system/node/` is a directory rule (due to the trailing `/`) so it's not redundant with `/sys/devices/system/node/*/meminfo r,`